store/urls.py

```diff
@@ -2,7 +2,7 @@
 from django.conf import settings
 from django.conf.urls.static import static
 from . import views
-from .views import BookDetailView, HomeView ,CategoryDetailView , RegistrationView , LoginView , AddToCartView , CartView, QuantityView, RemoveView
+from .views import UpdateBook,OrderView, ReceivedOrdersView, RemoveBook, BookDetailView, HomeView, ProfileView ,AddBook, CategoryDetailView , RegistrationView , LoginView , SearchView , AddToCartView,  CartView, QuantityView, RemoveView
```

Comment:
I think you are not using black and isort for code formating. please do format code before push on branch.

Reply:
sure sir